docs/Changelog.md

```diff
@@ -5475,6 +5519,50 @@ opset_import {
 <dd>Constrain input and output types to float tensors.</dd>
 </dl>
 
+### <a name="Unique-1"></a>**Unique-1**</a>
+
+  Returns the unique scalar elements of the input tensor as a 1-D tensor.
+
+#### Versioning
+
+This operator is used if you are using version 1 of the default ONNX operator set until the next BC-breaking change to this operator; e.g., it will be used if your protobuf has:
+
+~~~~
+opset_import {
+  version = 1
+}
+~~~~
+
+#### Attributes
+
+<dl>
+<dt><tt>return_inverse</tt> : int</dt>
+<dd>(bool) Whether to also return the indices for where elements in the original input ended up in the returned unique list.</dd>
+</dl>
+
+#### Inputs
+
+<dl>
+<dt><tt>X</tt> : T</dt>
+<dd>Tensor to find unique elements of</dd>
+</dl>
+
+#### Outputs
+
+<dl>
+<dt><tt>Y</tt> : T</dt>
+<dd>1-D tensor containing unique elements of the input tensor.</dd>
```

Comment:
We should either specify the order or explicitly state that the order of the elements is undefined.